### PR TITLE
KVStore: Enhance o11y of large txn

### PR DESCRIPTION
**Pull request by CalvinNeo**
### What problem does this PR solve?

Issue Number: ref #8736

Problem Summary:

Record delta of large txn delta keys.
A lock for a large txn lock cf key will have generation > 0.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by CalvinNeo** [comment]
image

File: dbms/src/Storages/KVStore/TiKVHelpers/DecodedLockCFValue.h [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#pragma once
+
+#include <Common/Exception.h>
+#include <Core/Types.h>
+#include <Storages/KVStore/Decode/DecodedTiKVKeyValue.h>
+
+namespace DB
+{
+namespace RecordKVFormat
+{
```
Comment (JaySon-Huang) [inline dbms/src/Storages/KVStore/TiKVHelpers/DecodedLockCFValue.h:24]:
```suggestion
namespace DB::RecordKVFormat
{
```

File: dbms/src/Storages/KVStore/TiKVHelpers/DecodedLockCFValue.h [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    bool use_async_commit{0};
+    UInt64 min_commit_ts{0};
+    std::string_view secondaries;
+    std::string_view primary_lock;
+    // For large txn, generation is not zero.
+    UInt64 generation{0};
+};
+
+} // namespace RecordKVFormat
+} // namespace DB
```
Comment (JaySon-Huang) [inline dbms/src/Storages/KVStore/TiKVHelpers/DecodedLockCFValue.h:49]:
```suggestion
} // namespace RecordKVFormat
} // namespace DB
```
```suggestion
} // namespace DB::RecordKVFormat
```

**Review by JaySon-Huang — APPROVED** [review]
LGTM with minor comment

**Review by Lloyd-Pottiger — APPROVED** [review]

**Pull request merged**